### PR TITLE
Customisations and Enhancements

### PR DESCRIPTION
**Pull request by rajarju**
## Description
Enhancements
- Configure custom web-src directory
- Skip serve flag to skip building and serving ui with parcel on local development
- Multiple entry point support for parcel. Parcel will consider all *.html files under web-src as entry file

## Related Issue

#310 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by shazron — APPROVED** [review]

**Comment by shazron** [comment]
@rajarju @purplecabbage unfortunately after further testing `aio app run`  doesn't work with this commit. I can't connect to the server at https://localhost:9080

**Comment by shazron** [comment]
Screen Shot 2020-11-17 at 11 20 32 AM

File: src/lib/runDev.js [hunk 1]
```diff
@@ -179,37 +183,38 @@ async function runDev (args = [], config, options = {}, log = () => {}) {
...
-        outDir: devConfig.web.distDev,
-        contentHash: false,
-        watch: true,
-        minify: false,
-        logLevel: 1,
-        ...bundleOptions
-      }
+      if (!options.skipServe) {
+        log('starting local frontend server ..')
+        const entryFile = path.join(devConfig.web.src, '*.html')
```
Comment (shazron) [inline src/lib/runDev.js:188]: this is the line that causes the 404 @rajarju @purplecabbage I will revert to index.html

Comment (shazron) [reply src/lib/runDev.js:188]: Of course https://localhost:9080/index.html works but this will break current workflows

Comment (shazron) [reply src/lib/runDev.js:188]: Parcel issue: https://github.com/parcel-bundler/parcel/issues/1315 
(not sure if I understand it fully)

Comment (shazron) [reply src/lib/runDev.js:188]: Option added in parcel-bundler@2. we are using parcel-bundler@1
https://github.com/parcel-bundler/parcel/pull/3996

Comment (shazron) [reply src/lib/runDev.js:188]: Applying PR https://github.com/adobe/aio-cli-plugin-app/pull/324.
The entrypoint fix should go in another PR (not sure what the solution there is, besides updating parcel-bundler)

Comment (rajarju) [reply src/lib/runDev.js:188]: @shazron Thanks for reverting this.

**Pull request merged**